protocol/test/package-lock.json

```diff
@@ -2430,7 +2429,6 @@
                 "@types/node": "*",
                 "anymatch": "^3.0.3",
                 "fb-watchman": "^2.0.0",
-                "fsevents": "^2.3.2",
```

Comment:
Is modifying this file an issue? It was modified automatically after running `npm install`. @pierluca

Reply:
that's okay. @Xelowak should now be able to tell you all you ever want to know about package-lock.json :-P